# Identify Proteins Critical to Learning in a Mouse Model of Down Syndrome
This is 18F EECS E6690 Statistical Learning for Biological and Information Systems Final Project **Identify Proteins Critical to Learning in a Mouse Model of Down Syndrome**.

### Original Paper
[Identify Proteins Critical to Learning in a Mouse Model of Down Syndrome](https://www.ncbi.nlm.nih.gov/pubmed/26111164)   
Higuera C, Gardiner KJ, Cios KJ (2015) Self-Organizing Feature Maps Identify Proteins Critical to Learning in a Mouse Model of Down Syndrome. PLoS ONE 10(6): e0129126. doi:10.1371/journal.pone.0129126

> Down syndrome (DS) is a chromosomal abnormality (trisomy of human chromosome 21) associated with intellectual disability and affecting approximately one in 1000 live births worldwide. The overexpression of genes encoded by the extra copy of a normal chromosome in DS is believed to be sufficient to perturb normal pathways and normal responses to stimulation, causing learning and memory deficits. In this work, we have designed a strategy based on the unsupervised clustering method, Self Organizing Maps (SOM), to identify biologically important differences in protein levels in mice exposed to context fear conditioning (CFC). We analyzed expression levels of 77 proteins obtained from normal genotype control mice and from their trisomic littermates (Ts65Dn) both with and without treatment with the drug memantine. Control mice learn successfully while the trisomic mice fail, unless they are first treated with the drug, which rescues their learning ability. The SOM approach identified reduced subsets of proteins predicted to make the most critical contributions to normal learning, to failed learning and rescued learning, and provides a visual representation of the data that allows the user to extract patterns that may underlie novel biological responses to the different kinds of learning and the response to memantine. Results suggest that the application of SOM to new experimental data sets of complex protein profiles can be used to identify common critical protein responses, which in turn may aid in identifying potentially more effective drug targets.

### What we did
Reproduced part of experiments in the paper, and tried different techniques but without biological interpretation.

### Authors
Di Wu (dw2834@columbia.edu), Jiliang Ma and [Qinyuan Wei](https://github.com/davidwei001)
